parm/fv3lam_post_avblflds.xml

```diff
@@ -1110,8 +1110,8 @@
 
     <param>
        <post_avblfldidx>124</post_avblfldidx>
-       <shortname>CLWMR_ON_HYBRID_LVL</shortname>
```

Comment:
@EricJames-NOAA We unified avblds.xml versions and would like to use post_avblflds.xml for all developing applications.  We plan to phase out legacy avblds.xml version e.g. fv3lam_post_avblflds.xml, nam_post_avblflds.xml in the future. Could you add all your changes in post_avblflds.xml instead of fv3lam_post_avblflds.xml and use post_avblflds.xml for flat text file generation?

Reply:
@WenMeng-NOAA thanks for the comment.  Yes, it sounds like a good idea to unify avblflds.xml.  I will test this for our GSL RRFS.  Regarding differences between EMC and GSL RRFS, I don't know much about the EMC RRFS.  However I do know that EMC RRFS uses 4-level NOAH LSM, while GSL RRFS uses 9-level RUC LSM.   So we will need to have slightly different fv3lam.xml files.

Reply:
@EricJames-NOAA It would be fine for UPP keeping two control files for RRFS, one for EMC RRFS, one for GSL RRF. We would like post_avblflds.xml being the only avblflds.xml for all developing applications. Thanks!